src/components/Footer.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+
+// MUI UI/UX styles
+import Box from "@mui/material/Box";
+import Container from "@mui/material/Container";
+import Grid from "@mui/material/Grid";
+import Link from "@mui/material/Link";
+import { ImageListItem } from "@mui/material";
```

Comment:
import ImageListItem from "@mui/material/ImageListItem", so the import style will be consistency